### DISCUSSION TITLE
Odokumenterat "badplatsområde" i Havs- och vattenmyndighetens Badplatsen publikt API (testLocationProfile)

### DISCUSSION THREAD

**Discussion by soleildeminuit** [salgo60/Svenskabadplatser#29] [General]
Polygonen **badplatsomrade** är inkapslad i en **bpomrwrapper**
Ex. https://badplatsen.havochvatten.se/badplatsen/api/testlocationprofile/SE0441233000000026

Jag ser ingen versionshantering i API:et eller dokumentationen, så jag vet inte när **bpomrwrapper** tillkom.

**Comment by salgo60** [comment]
Spännande ytor vore trevliga... 

Efter att ha varit inne i Open Street Map och ibland även ritat stranden så skulle man vilja ha en mer auktoriserad källa och kanske ha objekt som strand, swimming area, koordinat mätplats, [angränsande riskområden](https://www.lysekilsposten.se/artikel/lysekilsbad-kan-vara-fororenade-2/) verkar som dagens mätningar har en del som enligt denna [artikel](https://www.lysekilsposten.se/artikel/lysekilsbad-kan-vara-fororenade-2/) borde förbättras....

Andra saker som kan förbättras, toaletter, ramper.... som geodata. [Tillgänglighetsdatabasen](https://www.t-d.se/) känns rätt men känns mest som text som skall läsas där kanske geo objekt passar bättre (gissar att dom inte svarar på fråga är en indikation på ett projekt som ger upp...) rader som "Arvsfonden har genom sitt ekonomiska stöd möjliggjort TD:s utveckling." brukar vara en indikation att saker "ruttnar bort" när projektet avslutats eller skall bli projekt på drift....  plus har kommuner svårt att beskriva en badplats med 7 attribut så är steget ganska långt till att lägga sig på en detaljnivå som kan vara av intresse för funktionshindrade gissar jag

* vi kanske skall vara med på [utlysningen ;-)](http://www.naturvardsverket.se/Stod-i-miljoarbetet/For-forskare-och-granskare/Miljoforskningsanslaget/Oppna-utlysningar/Forskningsmedel-om-digitalisering-som-stod-for-en-hallbar-forvaltning-/)

**Comment by salgo60** [comment]
Lite Off Topic men får vi ytor så har jag funderat på att ha dessa ytor i [iNaturalist](https://www.inaturalist.org/) dvs. en app för oss dödliga att observera djur och blommor [länk](https://forum.inaturalist.org/t/creating-inaturalist-places-and-linking-to-wikidata-using-geojson/12220) 

* idag finns 3 egenskaper i Wikidata för iNaturalist
  * [P3151](https://www.wikidata.org/wiki/Property_talk:P3151) iNaturalist taxon-ID har över 580 000 kopplingar
  * [P7471](https://www.wikidata.org/wiki/Property_talk:P7471) iNaturalist plats-ID dvs, motsvarande instagram plats-id men här är det en yta för att samla ihop observationer... eftersom Naturvårdsverket slåppt alla sina naturreservat som [CC-0](https://creativecommons.org/share-your-work/public-domain/cc0/) så vore det en bra kandidat att även ha i iNaturalist
  * [P5683](https://www.wikidata.org/wiki/Property_talk:P5683) iNaturalist observation ID 
  
  Fick Södertälje kommun att lägga till samma som för Naturreservat tidigare se [blog](http://minancestry.blogspot.com/2020/05/blog-post.html) där vi även testade att skapa yta i iNaturalist
  
Idag finns det folk som letar efter CC-0 bilder i iNaturalist för arter som saknar bild i Wikidata och tankar ned dom. Tror det var 50 000 arter som saknar bild se [video](https://www.youtube.com/watch?v=zpHsHwR_kBc)  

* lista kopplingar idag [Sverige <-> Naturreservat <-> iNaturalist](https://w.wiki/3L3f)
  * exempel [Lida](https://www.inaturalist.org/places/154087) mer i [video](https://youtu.be/1Gy2MLAALwc)
  
  
![image](https://user-images.githubusercontent.com/14206509/118354360-03835980-b56b-11eb-9c6b-1d4090460302.png)

**Comment by soleildeminuit** [comment]
Example of test location area (**badplatsomrade**)
![testLocationArea](https://user-images.githubusercontent.com/10654578/118363896-b917d200-b596-11eb-8fcb-60d0933b6100.PNG)

**Comment by salgo60** [comment]
Snyggt 

* En grej som jag skulle vilja ha tydligare från havsmyndigheten att deras data är CC-0 nu fick jag ett halv svar….
* kan man skapa geojson av detta kan vi ha det som naturreservaten finns på Wikicommons är min gissning

Exempel karta https://map.wikilovesearth.se/#59.22133,18.09131,13z <https://map.wikilovesearth.se/#59.22133,18.09131,13z> där Kalle som laddade upp Geojson hämtat ut naturreservat
* Orlångens naturreservat https://www.wikidata.org/wiki/Q10611591	 <https://www.wikidata.org/wiki/Q10611591>
	* Property:P3896 Geografisk form = https://commons.wikimedia.org/wiki/Data:/Sweden/Nature_reserves/2020/Orl%C3%A5ngen/2001393.map <https://commons.wikimedia.org/wiki/Data:/Sweden/Nature_reserves/2020/Orl%C3%A5ngen/2001393.map>
		* action=raw <https://commons.wikimedia.org/w/index.php?title=Data:/Sweden/Nature_reserves/2020/Orl%C3%A5ngen/2001393.map&action=raw>

**Reply by salgo60** [reply]
@anderselias  Den som letar skall hitta ;-)

* [testlocation_areas.geojson](https://github.com/salgo60/Svenskabadplatser/blob/main/data/testlocation_areas.geojson)
* [testlocation_points.geojson](https://github.com/salgo60/Svenskabadplatser/blob/main/data/testlocation_points.geojson)
